relayer/crates/starknet-chain-components/src/types/cosmos/timestamp.rs

```diff
@@ -35,6 +35,6 @@ where
         buffer: &mut Encoding::DecodeBuffer<'a>,
     ) -> Result<Timestamp, Encoding::Error> {
         let product![unix_secs] = encoding.decode_mut(buffer)?;
-        Ok(Timestamp::from_nanoseconds(unix_secs * 1_000_000_000))
+        Ok(Timestamp::from_nanoseconds(unix_secs))
```

Comment:
here as well.

Reply:
Opened https://github.com/informalsystems/ibc-starknet/pull/352 to fix this